### ISSUE TITLE
provided branch name is skipped or not read

### ISSUE THREAD

**Issue by mjmohammadi1** [jpb06/jest-badges-action#53]
Hey there, 
Thanks for your great work on building this action. I'm trying to use it to add coverage badges to my project, and getting the following error.

```
Error: 🔶 Unable to get current branch from github event.
🔶 Current branch does not belong to the branches allowed for badges generation, task dropped.
```
I'm not sure if this is because of my branch naming pattern, or something else.

This is how I'm using the action:
```
    - name: Generating coverage badges
      uses: jpb06/jest-badges-action@latest
      with:
        branches: chore/badges,main
```
I'd appreciate if you let me know how I can resolve this.

**Comment by jpb06** [comment]
Hello!
Sorry for getting back to you that late. 

I think the issue arises when we use `/` in branches names. I will take a look.

**Comment by jpb06** [comment]
https://github.com/jpb06/jest-badges-action/pull/54

**Comment by jpb06** [comment]
[Released as v1.9.1.4](https://github.com/jpb06/jest-badges-action/releases/tag/v1.9.14).

Please let me know how it rolls 🙇🏻

**Comment by mjmohammadi1** [comment]
> [Released as v1.9.1.4](https://github.com/jpb06/jest-badges-action/releases/tag/v1.9.14).
> 
> Please let me know how it rolls 🙇🏻

Works great, thanks for the quick fix
